netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -775,9 +879,9 @@ _net_ebpf_ext_compare_destination_address(_In_ const bpf_sock_addr_t* addr1, _In
 
 static void
 _net_ebpf_extension_connection_context_initialize(
+    _In_ uint64_t transport_endpoint_handle,
```

Comment:
```suggestion
    uint64_t transport_endpoint_handle,
```
`_In_` is only for pointer types.  Don't use it on value types unless it's required to match a broken prototype in a windows .h file.